src/components/common/Directors.vue

```diff
@@ -728,6 +728,11 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
    */
   @Prop({ default: () => [] }) readonly directors!: DirectorIF[]
 
+  /**
+   * Change of Director date from parent page, used to set our 'asOfDate' when a new date is selected.
+   */
+  @Prop({ default: '' }) readonly codDate!: string
```

Comment:
Won't this clash with `asOfDate` that is passed into getOrigDirectors()? What's the difference between codDate and asOfDate?

Maybe getOrigDirectors() just needs to be called at the right time?